helm_deploy/hmpps-visits-internal-admin-ui/values.yaml

```diff
@@ -1,6 +1,7 @@
 generic-service:
   nameOverride: hmpps-visits-internal-admin-ui
-
+  serviceAccountName: visit-someone-in-prison
```

Comment:
Do we need this line (`serviceAccountName: visit-someone-in-prison`) - tbh I don't know what it's for?

Reply:
as discussed removed serviceAccountName since its  something needed for IRSA changes and not part of this story.